include/NovelRT/Utilities/Misc.h

```diff
@@ -60,6 +60,25 @@ namespace NovelRT::Utilities
 
             return targetPtrs;
         }
+
+        static std::vector<std::string> SplitString(const std::string& targetString,
+                                                    const std::string& delimiter) noexcept
+        {
+            std::vector<std::string> returnVec{};
+            size_t start = 0;
+            size_t end = targetString.find(delimiter);
+
+            while (end != std::string::npos)
+            {
+                returnVec.emplace_back(targetString.substr(start, end - start));
+                start = end + delimiter.size();
+                end = targetString.find(delimiter, start);
+            }
+
+            returnVec.emplace_back(targetString.substr(start)); // THIS LINE ANGERS ME
+
+            return returnVec;
```

Comment:
is this safe just because it's from a static method? (Meaning that we don't lose it on the stack)

Reply:
yes

Reply:
why wouldn't it be safe? a vector is not a span that's having its underlying memory being yanked. Aren't we already doing this in other utility methods?

Reply:
I just forgot that this is copying value and not returning a ref 🙃 sorry about that!